server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -32,6 +32,8 @@ import {
 } from "../services";
 import { containsPathTraversal, parseToken } from "../utils";
 
+const MAX_TOKEN_LENGTH = 1000; // Maximum allowed token length in characters
```

Comment:
Curious what is the normal length of token?

Reply:
A reasonable token size is somewhat around 550, so we are keeping some buffer and rejecting tokens > 1000 length.